docs/v2/articles/indexing.md

```diff
@@ -116,9 +116,10 @@ myIndex.IndexItem(new ValueSet(
 Be default all indexing is done asynchronously. If you need to run indexing synchronously you should create a synchronous scope. This is for instance a necessary step for unit tests.
 
 ```cs
-using (myIndex.ProcessNonAsync())
+using (var luceneIndex = (myIndex as LuceneIndex))
```

Comment:
do we want to use `as` syntax? since this will return null if it is not that type, maybe just direct cast since that is what were expecting (LuceneIndex)myIndex else we can end up with null ref exception.

Reply:
Sure, fixed